blocks/editable/style.scss

```diff
@@ -3,6 +3,7 @@
 }
 
 .blocks-editable__tinymce {
+
```

Comment:
What's the guideline that directs this blank line here? Throughout the changes I'm inferring a guideline of "always add a newline before nested selector". Is that documented somewhere? Looks a bit out-of-place in some cases, particularly in contrast to non-nested selector styles not needing the new line.